aiothrift/pool.py

```diff
@@ -44,7 +55,7 @@ class ThriftPool:
     """Thrift connection pool.
     """
 
-    def __init__(self, service, address, *, minsize, maxsize, timeout=None):
+    def __init__(self, service, address, *, minsize, maxsize, timeout=None, framed):
```

Comment:
positional argument `framed` is not valid after keyword argument `timeout` .

Reply:
Fixed